homu/server.py

```diff
@@ -1,7 +1,8 @@
 import hmac
 import json
 import urllib.parse
-from .main import PullReqState, parse_commands, db_query, INTERRUPTED_BY_HOMU_RE, synchronize
+from .main import (PullReqState, parse_commands, db_query,
+                   INTERRUPTED_BY_HOMU_RE, synchronize)
```

Comment:
I much prefer block style indents (with trailing commas) in general, e.g.
```
from .main import (
    PullReqState,
    parse_commands,
    db_query,
    INTERRUPTED_BY_HOMU_RE,
    synchronize,
)
```
They take up more vertical space but I find them easier to read and cause much less diff noise, which is valuable for `git log`, `git blame` and friends.

This also works for method calls, dictionary and list literals, etc. in addition to imports.